pkg/codegen/python/gen.go

```diff
@@ -2142,12 +2143,14 @@ func genPackageMetadata(
 		}
 		fmt.Fprintf(w, "',\n")
 	}
-	if pkg.Homepage != "" {
-		fmt.Fprintf(w, "      url='%s',\n", pkg.Homepage)
+	urls := mapURLs(pkg)
```

Comment:
Kinda feel this was nicer just referring to the Homepage and Repository fields directly.  What are we gaining by putting them in a map and then looking them up with fixed strings?